backend/search.py

```diff
@@ -43,3 +44,49 @@ def search():
 
     results_json = results.exclude('image').to_json()
     return results_json
+
+
+@search_blueprint.route("/search_gps", methods=['GET'])
+def search_gps():
+    lat = float(request.args.get('lat', default=0))
+    lon = float(request.args.get('long', default=0))
+    miles = request.args.get('miles', default=10)
+
+    items = model.Item.objects().only('stores')
+
+    seen = set()
+    list_of_stores = []
+    for item in items:
+        for store in item.stores:
+            store_lat = store['location']['lat']
+            store_long = store['location']['long']
+            store_entry = {
+                'name': store['name'],
+                'lat': store_lat,
+                'long': store_long
+            }
+            t = tuple(store_entry.items())
+            if t not in seen:
+                seen.add(t)
+                list_of_stores.append(store_entry)
+
+    list_of_stores_gps = []
+    for store in list_of_stores:
+        r = 3963.0
```

Comment:
Please rename this to earth_radius so it's clear what it is.

Reply:
Done